forge-gui/res/cardsfolder/upcoming/ratonhnhake_ton.txt

```diff
@@ -0,0 +1,13 @@
+Name:Ratonhnhaké:ton

```

Comment:
does this display fine?

Reply:
Circles of Protection also use colon in name field, so it should be fine.